parse.py

```diff
@@ -40,9 +40,7 @@
     extensions = [ext for ext in extensions if ext not in targets]
     print(f"Extensions selected : {extensions}")
 
-    include_pseudo = False
-    if "-pseudo" in sys.argv[1:]:
-        include_pseudo = True
+    include_pseudo = "-pseudo" in sys.argv[1:]
```

Comment:
I agree to the concise approach being followed here to reduce the number of lines of codes. Given that we should focus on optimizing the codes as much as possible, but at the same time, we should not forget its maintainability and readabaility.

Following, **_"Explicit is better than Implicit"_**: the core tenet of Python, the already existing codes explicitly communicates the default state for the inclusion of `pseudo instructions` as `(False)` and only sets it to `True `conditionally, making the flow of logic clearer. Readers can follow the intent more easily than inferring the meaning from a single line assignment. 

Additionally, I am tagging the discussions for reference regarding `pseudo instructions` inclusion for more information as to why we are using flags for explicit behaviour : #286 and #289

Reply:
I can't say I agree here. This isn't more explicit; it's just more verbose.

This is standard best practice. See [this blog for example](https://adamj.eu/tech/2020/01/17/simplify-your-ifs-that-return-booleans/#assignments).

In the long term it would be much better to use `argparse` of course, but this is a pretty straightforward simplification.